paket.dependencies

```diff
@@ -1,35 +1,59 @@
 source http://nuget.org/api/v2
+redirects: on
+strategy: max
 
-nuget FSharp.Compiler.Service
 nuget Fantomas
+nuget FSharp.Compiler.Service
+nuget FSharp.Core
+nuget FSharp.Management
 nuget FSharp.ViewModule.Core
-nuget FsXaml.Wpf
-nuget SourceLink.SymbolStore 0.5.0
 nuget FSharpLint.Core prerelease
-nuget VSSDK.IDE.Deploy.14
-nuget FSharp.Management
 nuget FsPickler
-nuget FSharp.Core redirects:on
+nuget FsXaml.Wpf
+nuget SourceLink.SymbolStore 0.5.0
 
+group vssdklegacy
+    source http://nuget.org/api/v2
+    framework >= net40
+    
+    nuget Microsoft.VisualStudio.Shell.Interop.8.0
+    nuget Microsoft.VisualStudio.Shell.Immutable.10.0
+    nuget VSSDK.ComponentModelHost.11
+    nuget VSSDK.DTE
+    nuget VSSDK.DTE.8
+    nuget VSSDK.DTE.10
+    nuget VSSDK.NavigateTo
+    nuget VSSDK.VSLangProj
+    
+    
+group vssdk
+    source http://nuget.org/api/v2
+    framework >= net45
+    
+    nuget Microsoft.VisualStudio.Editor
+    nuget Microsoft.VisualStudio.Language.Intellisense
```

Comment:
@jaredpar thanks for feedback!

You can see all versions in https://github.com/fsprojects/VisualFSharpPowerTools/blob/master/paket.lock

I think with the qualified paths (not based on assembly name resolution but actual path to the file) to the assemblies in *proj files it should pick up the correct versions right?

We actually simplified with input of @dungpa by having only vssdk14 (+ legacy for EnvDTE etc.) but I'm not sure yet it allows to run our extension in VS2013.